tests/utils/test_file_utils.py

```diff
@@ -364,3 +365,22 @@ def test_shutil_copytree_without_file_permissions(tmp_path):
     assert set(os.listdir(dst_dir.joinpath("subdir"))) == {"subdir-file.txt"}
     assert dst_dir.joinpath("subdir/subdir-file.txt").read_text() == "testing 123"
     assert dst_dir.joinpath("top-level-file.txt").read_text() == "hi"
+
+
+def test_get_total_size(tmp_path):
+    root = str(tmp_path)
+
+    subdir = os.path.join(root, "subdir")
+    os.mkdir(subdir)
+    files = {
+        "file1.txt": "hello world",  # 11 bytes
```

Comment:
Ah yeah - this is also something I'd like to have your advice. Is it ok to save some example artifacts in the repo for testing purpose? For `transformers` it can be a hundred of MB. Is there a better practice we follow in mfllow?

Reply:
We definitely don't want to save anything like that in the repo. We always fetch such files from huggingface_hub if we need to validate. You can see examples of us downloading and using transformers models within the transformers unit test suite in MLflow (here's an example: https://github.com/mlflow/mlflow/blob/49415f37edc02da7c8e9ccee2b78e294fe5aa26e/tests/transformers/test_transformers_model_export.py#L980-L989 )

We should definitely not try to store such a large file(s) in the actual repo, though.

Reply:
Ahh yeah. I'll some more tests cases following this. I was also hesitated to introduce network calls, but I guess that's fine if we're already doing it.